src/background/responders.ts

```diff
@@ -64,10 +103,10 @@ export const responders: Responders<Action> = {
     return { auth: { state: 'authenticated', user: { photoUrl } } }
   },
   authenticationFailure(state, { error }): Partial<StoreState> {
-    return { alert: error.message, auth: { state: 'auth_failed' } }
+    return { alertHtml: error.message, auth: { state: 'not_authed' } }
```

Comment:
Why did you remove the `auth_failed` state?

Reply:
The `not_authed` state represents it well. You're not authed and not authenticating. We just need to also display an alert, which this does.